rule-template/cuevox.yaml

```diff
@@ -189,22 +189,24 @@ actions:
 
                   let commandToExecute = this.command;
 
-                  var executeFunction = function(parameter) {
+                  var sendCommand = function(parameter) {
                       if (!parameter || typeof(parameter) != "object") {
                           logger.info("Trying to send a command, but no proper object parameter found")
-                          return;
+                          return false;
                       }
 
                       if (!parameter.items) {
                           logger.info("Trying to send a command, but no items parameter found")
-                          return;
+                          return false;
                       }
 
                       parameter.items.forEach(item => {
                           item.sendCommand(commandToExecute);
                       });
-                  }
-                  return new EvaluationResult(true, result.remainingTokens, executeFunction, result.executeParameter);
+
+                      return true;
```

Comment:
I have modified the function signature used for the functions as part of the CMD expression to return if the function has executed successfully. This is why I have to mock the return value of the mocked functions as well.

I found without this a CMD expression that contained `null` arguments would otherwise still return `'status': 'success'` even though the executeFunction returns immediately.